File: jscomp/test/io_test.js
'use strict';


function f(param) do
  console.error("x");
  console.log(--[ () ]--0);
  console.log("hi");
  console.log(--[ () ]--0);
  return --[ () ]--0;
end

exports.f = f;
--[ No side effect ]--

File: jscomp/test/io_test.lua
__console = {log = print};


function f(param) do
  __console.error("x");
  __console.log(--[[ () ]]0);
  __console.log("hi");
  __console.log(--[[ () ]]0);
  return --[[ () ]]0;
end end

exports = {};
exports.f = f;
return exports;
--[[ No side effect ]]
